### PR TITLE
Added some missing translate tags for cards

### PR DESCRIPTION
**Pull request by brainexe**
Just a small PR to add some missing translation tags in the more-info part of following cards:
 - sun
 - script
 - updater

### CODE REVIEW

**Review by balloob — APPROVED** [review]

**Comment by balloob** [comment]
ok to merge when linting passes.

File: src/dialogs/more-info/controls/more-info-sun.js [hunk 1]
```diff
@@ -63,7 +63,11 @@ class MoreInfoSun extends LocalizeMixin(PolymerElement) {
   }
 
   itemCaption(type) {
-    return type === "ris" ? "Rising " : "Setting ";
+    if (type === "ris") {
+      return this.localize("ui.card.sun.rising");
+    } else {
```
Comment (zsarnett) [inline src/dialogs/more-info/controls/more-info-sun.js]: Remove the else. Just have the return under the if

Comment (brainexe) [reply src/dialogs/more-info/controls/more-info-sun.js]: done :)

File: translations/en.json [hunk 2]
```diff
@@ -768,7 +768,8 @@
                 "activate": "Activate"
             },
             "script": {
-                "execute": "Execute"
+                "execute": "Execute",
+                "last_action": "Last Action"
```
Comment (balloob) [inline translations/en.json]: This is the wrong place to add translations. it has to be in `src` dir.

Also, we shouldn't add translations for more info dialogs under the card key. Those are two different things.

**Review by balloob — CHANGES_REQUESTED** [review]

File: translations/en.json (hunk 2 above)
Comment (brainexe) [reply translations/en.json]: damn, right, fixed the wrong translation file.

But what translate key should be used here? All ~40 translation in the "more informaion dialogs" in found are located in "ui.card." translation context. Any suggestion?

Comment (balloob) [reply translations/en.json]: Let's create a new `more_info_control` under `dialogs`

Comment (balloob) [reply translations/en.json]: And then keyed off by domain

Comment (brainexe) [reply translations/en.json]: kk, should I also migrate the ~40 old ui.card.* usages inside the dialogs/more-info directory (might mark the translation as new in translate tool?!)? Or should it be part of another PR?

**Comment by pepeEL** [comment]
Please also add translation to this:
trans1

Card entity
card_entity

Glance card
card_entity button

**Comment by pepeEL** [comment]
Also add view editor:
add_view

add_view2

**Comment by pepeEL** [comment]
Right menu
right menu

**Comment by brainexe** [comment]
@balloob should I add more changes to this PR? 
I could also push another PR to add missing tags in lovelace code which were mentioned by @pepeEL?!

**Comment by pepeEL** [comment]
I think that good will be create new PR with this missing tags lovelace code.
But developer should accept it.

**Comment by pepeEL** [comment]
Ok sorry. I have good intentions and I did not want to offend anyone.

**Pull request merged**